src/transport/AdminPairingTable.cpp

```diff
@@ -27,41 +27,68 @@
 namespace chip {
 namespace Transport {
 
-CHIP_ERROR AdminPairingInfo::StoreIntoKVS(PersistentStorageDelegate & kvs)
+PersistentStorageDelegate * gStorage = nullptr;
+AdminPairingTableDelegate * gDelegate = nullptr;
```

Comment:
Shouldn't these be in an anonymous namespace? Or better yet be members of AdminPairingTable instead of globals?

Reply:
Mved it + added comments